src/Backend/Modules/Faq/Engine/Model.php

```diff
@@ -16,14 +20,14 @@ class Model
     const QUERY_DATAGRID_BROWSE =
         'SELECT i.id, i.category_id, i.question, i.hidden, i.sequence
          FROM faq_questions AS i
-         WHERE i.language = ? AND i.category_id = ?
+         WHERE i.locale = ? AND i.category_id = ?
```

Comment:
Has the field `category_id` still the same name? I would think it would be `categoryId`

Reply:
yes, doctrine does relationships with an underscore and other fields with camelcase

Reply:
kinda stupid...

Reply:
yeah...

Reply:
you can always overwrite that by passing `categoryId` as a name